install/kubernetes/helm/istio-remote/charts/security/templates/cleanup-secrets.yaml

```diff
@@ -87,7 +87,7 @@ spec:
       serviceAccountName: istio-cleanup-secrets-service-account
       containers:
         - name: hyperkube
-          image: "{{ .Values.global.hyperkube.hub }}/hyperkube:{{ .Values.global.hyperkube.tag }}"
+          image: "{{ .Values.global.hyperkube.repository }}:{{ .Values.global.hyperkube.tag }}"
```

Comment:
My concern with all these changes is e.g. quay.io/coreos/hyperkube is not a repository. We'd need to introduce an image parameter (overkill imo) and the current system works fine because an image name like "hyperkube" is highly unlikely to change (or in fact something we'd want to change), whereas repo and tag are things that we do change regularly.

Reply:
The avoidance of variable sprawl is why the last PR that hardcoded the image names was hardcoded :).  I we should ask ourselves, what problem are we solving and balance that with the general problem of too many variables in values.yaml.

Reply:
I'm +1 to add a new parameter `image` as follows:

```
hub: quay.io/coreos
image: hyperkube
tag: v1.7.6_coreos.0
```

@linsun @sdake there are indeed cases that customer want to customize the docker image names for their own helm chart, when customer build Istio into their own system, they may want to update the docker image name based on their system.

Reply:
wfm.

Reply:
I'm ok with hyperkube, but kiali and prometheus? At least let's keep those hardcoded.

Reply:
How about adding `image` for all but with default value, and end user can update based on their requirement with ease. @ostromart @sdake

Reply:
I don't believe changing kiali/prometheus name is a use case so why support it? It just increases the length of the params list.